auth.go

```diff
@@ -667,26 +663,26 @@ func (x *Central) ArchiveIdentity(userId UserId) error {
 	return nil
 }
 
-// Get userid from identity.
-func (x *Central) GetUserIdFromIdentity(identity string) (UserId, error) {
-	userId, e := x.userStore.GetUserIdFromIdentity(identity)
+// Get AuthUser object from identity.
+func (x *Central) GetUserForIdentity(identity string) (AuthUser, error) {
```

Comment:
The use of the word "For" here is inconsistent, and generally doesn't comprehend very well. Rather use "From". Same for GetUserForUserId. Eg `GetUserFromIdentity`, `GetUserFromId`